### ISSUE TITLE
About the training data for test submission 

### ISSUE THREAD

**Issue by zhangyp15** [astra-vision/MonoScene#54]
Hi, I have a question about the test submission for semantic kitti. For test submission, should we retrain the model with train+val sequences or directly use the trained model for validation ? 

Thanks!

**Comment by anhquancao** [comment]
Hi @zhangyp15, thank you for your interest in MonoScene. We only use the train set for training and select the model with the highest mIoU on the validation set. You can also train on train+val as long as you do the same for all the methods you compare with.

**Comment by zhangyp15** [comment]
Many thanks for your quick reply!
